dydx3/modules/private.py

```diff
@@ -550,7 +550,7 @@ def cancel_all_orders(
             "LINK-USD",
         ]
 
-        :returns: None
+        :returns: Array of orders
```

Comment:
(Ideally we should be indicating the actual structure of the returned data, i.e. a dict with key `cancelOrders`.. but since this is how the other endpoints are documented this is ok for this PR)